### ISSUE TITLE
Reformat bulk upload guide with real elements

### ISSUE THREAD

**Issue by lauraykerr** [CDCgov/prime-simplereport#4003]
## Area or Component
Bulk Person Upload Guide

## Quick Summary
Ensure content in the bulk  guide is formatted with proper headers, lists, etc. 

## Accessibility Violation
PDFs should be formatted with real headers, lists, and tables at the source document, resulting in a properly tagged PDF that can be read by screen readers.

## Action Requested
Add proper headers and list elements to the source doc. 

## Source Code (optional)
[Bulk person upload guide - 07082022](https://docs.google.com/document/d/1OWKg4zUtMW6eAD297DIo70O8-SFoKlA4yPAD389eAI0/edit#)
